Feat/introduction

jyng opened the pull request.

urcades committed:
Added replacement svg assets, deleted ones we will no longer use

urcades committed:
Create technology.png

jyng committed:
introduction layout complete

jyng committed:
Merge branch 'next-slim' of github.com:urbit/urbit.org into feat/introduction

urcades committed:
+ add gray backgrounds to <pre> code for legibility

jyng committed:
Merge branch 'feat/introduction' of github.com:urbit/urbit.org into feat/introduction

jyng committed:
adding pictures and text

jyng committed:
spacing changes

urcades committed:
add images

urcades committed:
reference newly added images

jyng committed:
Merge branch 'next-slim' into feat/introduction

urcades commented:
@g-a-v-i-n @matildepark do either of ya have any idea why our checks might be failing at this point? I'm getting a possibly related error while trying to start up zola:

<img width="609" alt="Screen Shot 2019-10-10 at 8 22 56 PM" src="https://user-images.githubusercontent.com/1195363/66616263-0bfcfc80-eb9e-11e9-93ce-c619efe56663.png">


It's kind of crazy because neither I nor @jyng touched anything in docs afaik

matildepark commented:
Working with Jimmy on it out of band. As with Gavin’s PR, it comes from a relative link not existing. I committed to docs a few hours ago and pulled docs into the branch. So merge next-slim back in. 

Sent from my iPhone

> On Oct 10, 2019, at 8:39 PM, é <notifications@github.com> wrote:
> 
> ﻿
> @g-a-v-i-n @matildepark do either of ya have any idea why our checks might be failing at this point? I'm getting a possibly related error while trying to start up zola:
> 
> 
> 
> It's kind of crazy because neither I nor @jyng touched anything in docs afaik
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub, or unsubscribe.

jyng committed:
using docs master

urcades committed:
updated svgs

urcades committed:
refactoring the naming scheme for svgs

urcades committed:
replacing archive .svg

urcades committed:
normalized image sizing per the design (50% text, 50% figures), removed sidebar nav

urcades committed:
Fixed paged header styling, removed large type/thin weight

urcades committed:
Removing the 30em max width on essays, additional info in description

I noticed that the essay content for Understanding Urbit had a noticeably smaller max width than what we were displaying in "Understanding Urbit: Introduction", and saw that essay-wide, across the entire site, all non-image non-figure <article> widths were constrained to 30em max width, which is contrary to the original intent of the design — Essentially, the baseline rule is that ALL (and I mean, literallyl, all) primary body content on any given page should occupy the middle 6 columns (on a large enough viewport), regardless of where they appear. This means text, no matter which page it appears on, should always be set to occupy these columns at any given time no matter what content is displayed alongside the text.

I get the sense this removal of the 30em max width might break some other pages, which might suck but is neccesary so that pages retain this strong uniformity throughout the docs + homepages

urcades committed:
Accepting current change to render both Roadmap and History diagrams.

jyng committed:
mobile layout and responsive spacing

jyng committed:
Merge branch 'next-slim' of github.com:urbit/urbit.org into feat/introduction

matildepark commented on templates/understanding-urbit/list.html:
```diff
@@ -8,65 +8,39 @@
   <a href="/understanding-urbit/">
     Understanding Urbit
   </a>
-  {% include "partials/navigation-search.html" %}
```
Did you intend to remove the Search from the sidebar on UU's intro, as well? And the sidebar is on the other pages — if you want to remove it from all of them, you'll have to do that again in `/templates/understanding-urbit/post.html`

matildepark reviewed (approved):
Otherwise this is good to go.

jyng commented:
its intentional for introduction to not have a sidebar so people can focus on the page. LGTM.
